src/Microsoft.Data/BaseColumn.cs

```diff
@@ -173,6 +190,10 @@ public void SetName(string newName, DataFrame dataFrame = null)
         /// </summary>
         public virtual DataFrame Description() => throw new NotImplementedException();
 
+        public override bool Equals(object obj) => ReferenceEquals(this, obj);
+
+        public override int GetHashCode() => base.GetHashCode();
```

Comment:
Why are these Equals and GetHashCode changes necessary for `AppendRow and to fix NullCount bugs`?